### ISSUE TITLE
the other way - DUN from Android

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mmorciegov/android-wifi-tether#599]
```
Which device (manufacturer, type)?
All Android

Which firmware is installed?
All

What version of wireless tether are you using?
-

What steps will reproduce the problem?
1. Want to make an outbound Bluetooth Dial UP connection using another phone 
with BT.
2.
3.

What is the expected output? What do you see instead?

Successfull connection

Please provide any additional information below.

I want to ask if the other way around is possible.
In short, I'd like to make an outbound BT DUN connection from my Android phone 
using another phone (nokia) as a BT modem. would that be possible ?
There is a software at this URL which does what I want but I couldn't get it to 
work:
http://forum.xda-developers.com/showthread.php?t=770259

```

Original issue reported on code.google.com by `filizakt...@gmail.com` on 20 Sep 2010 at 9:20
